### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by sofipacifico**
* Fix spelling 
* Validate before to show a final pop-up, if there are… invalid or empty calcifications

### CODE REVIEW

**Comment by andresmoschini** [comment]
Looks fine

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 1]
```diff
@@ -86,20 +86,29 @@
...
-            modalViewModel.title("Finalizar evaluación");
-            modalViewModel.text("¿Desea finalizar la evaluación con calificaciones vacías?");
-            modalViewModel.buttonBackText("Cancelar");
-            modalViewModel.buttonConfirmText("Confirmar");
-            modalViewModel.show(true);
-            modalViewModel.isConfirmButtonVisible(true);
-            modalViewModel.isFinalButtonVisible(false);
-        }
-        else {
+        if (!this.isValid() || this.hasEmptyValues()) {
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: I think that this nested if could be rewritten as:

```
 if (!this.isValid()) {
    ...
} else if (this.hasEmptyValues()) {
    ...
} else {
    ...
}
```

Comment (sofipacifico) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: :+1:

**Comment by joseguti84** [comment]
I will refactor this code and I will make a new PR.

**Pull request closed without merging**